plugins/Rockets/jsonPropertyMap.h

```diff
@@ -552,25 +757,104 @@ inline bool from_json(brayns::PropertyMap& obj, const std::string& json)
             return false;
         switch (obj.getPropertyType(propName))
         {
-        case PropertyMap::Property::Type::Float:
-            SET_PROPERTY_NUMBER(Double)
+        case PropertyMap::Property::Type::Double:
+        {
+            double val;
+            if (!get_property(m.value, val))
+                return false;
+            obj.updateProperty(propName, val);
+            break;
+        }
         case PropertyMap::Property::Type::Int:
-            SET_PROPERTY(Int)
+            if (!parseSetProperty<int32_t>(m.value, obj, propName))
+                return false;
+            break;
         case PropertyMap::Property::Type::String:
-            SET_PROPERTY(String)
+            if (!parseSetProperty<std::string>(m.value, obj, propName))
+                return false;
+            break;
         case PropertyMap::Property::Type::Bool:
-            SET_PROPERTY(Bool)
-        case PropertyMap::Property::Type::Vec2f:
-            SET_ARRAY(double, Double, 2)
+            if (!parseSetProperty<bool>(m.value, obj, propName))
```

Comment:
Shouldn't it be like the code for double?

Reply:
done.